frontend/clueless.js

```diff
@@ -14,37 +14,30 @@ function establishWebsocket() {
       //webpage will update status from Connecting to Connected
       document.getElementById('status').innerHTML = 'Connected';
       // connection is opened and ready to use
+
+      connection.send(JSON.stringify({eventType:'TEST'}));
+
+      // send JOIN_REQUEST
+      var tag = prompt("Please enter your name");
+      var joinRequest = {eventType: "JOIN_REQUEST", playerTag: tag };
+      connection.send(JSON.stringify(joinRequest));
     };
 
     connection.onerror = function (error) {
       console.log('websocket messed up');
         // an error occurred when sending/receiving data
     };
 
-    connection.onmessage = function (message) {
-        // try to decode json (I assume that each message from server is json)
-        try {
```

Comment:
true we should. previously this was getting overridden anyways by the code on line 100.